tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -110,10 +134,13 @@ snd (a, b) = b
 // s2n state transition that incorporates both s2n_advance_message and
 // s2n_conn_set_handshake_type
 s2nTrans : connection -> connection
-s2nTrans conn = conn''
-          where conn' = if helloState then conn_set_handshake_type conn else conn
-                conn'' = advance_message conn'
+s2nTrans conn = conn3
+          where conn2 = if helloState then conn_set_handshake_type conn 
+                        else if noCertAuthState then conn_set_handshake_no_client_cert conn
+                        else conn
+                conn3 = advance_message conn2
                 helloState = ACTIVE_MESSAGE conn == CLIENT_HELLO \/ ACTIVE_MESSAGE conn == SERVER_HELLO
+                noCertAuthState = ACTIVE_MESSAGE conn == CLIENT_CERT /\ conn.no_client_cert /\ conn.actual_protocol_version == S2N_TLS13
```

Comment:
Put a comment on this specifically explaining that the TLS1.2 proof doesn't support no_client_cert atm. You could also link to the issue to fix that.